tests/testcases/100_check-k8s-conformance.yml

```diff
@@ -0,0 +1,27 @@
+---
+- hosts: kube-master[0]
+  vars:
+    sonobuoy_version: 0.14.1
+    sonobuoy_arch: amd64
+
+  tasks:
+  - name: Download sonobuoy
+    get_url:
+      url: "https://github.com/heptio/sonobuoy/releases/download/v{{ sonobuoy_version }}/sonobuoy_{{ sonobuoy_version }}_linux_{{ sonobuoy_arch }}.tar.gz"
+      dest: /tmp/sonobuoy.tar.gz
+
+  - name: Extract sonobuoy
+    unarchive:
+      src: /tmp/sonobuoy.tar.gz
+      dest: /usr/local/bin/
+      copy: no
+
+  - name: Run sonobuoy
+    command: /usr/local/bin/sonobuoy run --wait
```

Comment:
If you want to speed up the tests there’s the hidden option --e2e-parallel introduced [here](https://github.com/heptio/sonobuoy/pulls?utf8=✓&q=is%3Apr+is%3Aclosed+parallel)
We’ve this value set to 30 and the tests passed from 2 hours to 10 minutes.